src/runtimes/wapc.rs

```diff
@@ -44,6 +46,90 @@ pub(crate) fn host_callback(
                     }
                     Ok(Vec::new())
                 }
+                _ => {
+                    error!(namespace, operation, "unknown operation");
+                    Err(format!("unknown operation: {}", operation).into())
+                }
+            },
+            "oci" => match operation {
+                "manifest_digest" => {
+                    let policy_mapping = WAPC_POLICY_MAPPING.read().unwrap();
```

Comment:
What is the expectation with burrego based policies? Do we plan to support something like this on those?

Reply:
I don't know, maybe in the future... but right now there's no way for them to reach out to the host. The only way would be to have us define some new Rego built-in and then plug that into the `CallbackHandler`